main.go

```
@@ -0,0 +1,791 @@
+}
+
+// Progress tracking
+type MigrationProgress struct {
+	LastThreadID     int   `json:"last_thread_id"`
+	CompletedThreads []int `json:"completed_threads"`
+	FailedThreads    []int `json:"failed_threads"`
+	LastUpdated      int64 `json:"last_updated"`
+}
+
+// GitHub GraphQL mutations
+const createDiscussionMutation = `
```

golangci-lint (1.64.8)

const `createDiscussionMutation` is unused

(unused)